stackstorm-compose/README.md

```diff
@@ -0,0 +1,305 @@
+# StackStorm in Docker Compose
+
+This docker-compose is provided as a way to allow someone to "get up and running" quickly with StackStorm using Docker. It is not designed to be used in production, but rather a way to test out StackStorm and facilitate pack development.
+
+## TL;DR
+
+```shell
+git clone git@github.com:stackstorm/st2-dockerfiles
+cd st2-docker/stackstorm-compose
+docker-compose up -d
+docker-compose run st2api st2ctl reload --register-all  # you should only have to do this the first time
```

Comment:
When I run this command, it always shows other components ##### st2 components status ##### as failed. I've searched but still don't know why. It should be just a wrong warning log, everything seems to be ok I guess.

```shell
##### st2 components status #####
st2actionrunner is not running.
st2api PID: 1
st2stream is not running.
st2auth is not running.
st2garbagecollector is not running.
st2notifier is not running.
st2resultstracker is not running.
st2rulesengine is not running.
st2sensorcontainer is not running.
st2chatops is not running.
st2timersengine is not running.
st2workflowengine is not running.
st2scheduler is not running.
mistral-server is not running.
mistral.api is not running.
```

Reply:
OK I think its because its designed to work in a single-machine. [As in this link](https://github.com/StackStorm/st2/blob/88834125296b588f601cd964b56e44c9d111b720/st2common/bin/st2ctl#L196).

Reply:
I'll have to make note of that in the README.  As you saw, the only service running in st2api is st2api, but the st2ctl reload will attempt to do all of them. Alternatively, if there is a better way to get all of the action/sensor/rules/triggers/etc registered without the user having to do anything, I would be all for that.

Reply:
Also @tunayagci, thanks for all the feedback. I've been making this docker-compose for my team at work, and between you and @winem, finding all sorts of ways to make it better for everyone.